Packs/Base/Scripts/DBotPredictURLPhishing/DBotPredictURLPhishing.py

```diff
@@ -0,0 +1,585 @@
+import demistomock as demisto
+from CommonServerPython import *
+from CommonServerUserPython import *
+import urllib
+from typing import List, Dict, Tuple
+import pandas as pd
+import base64
+import requests
+import dill
+import copy
+import numpy as np
+from tldextract import TLDExtract
+requests.packages.urllib3.disable_warnings()
+
+dill.settings['recurse'] = True
+
+NO_FETCH_EXTRACT = TLDExtract(suffix_list_urls=None, cache_dir=False)
+
+OOB_MAJOR_VERSION_INFO_KEY = 'major'
+OOB_MINOR_VERSION_INFO_KEY = 'minor'
+MAJOR_VERSION = 0
```

Comment:
Just to make sure I understand - everytime we will update the model on the docker, you will also update this variable in the script?

Reply:
Each time we will deliver new model/change in code we will change the major. If user update the model we will change the minor